lua/astrocommunity/pack/yaml/init.lua

```diff
@@ -1,7 +1,8 @@
 return {
   {
     "b0o/SchemaStore.nvim",
-    lazy = true,
+    event = "User AstroFile",
+    ft = { "yaml", "json" },
```

Comment:
```suggestion
    lazy = true,
```

this should be reverted. This is more performant as is. It is lazy loaded based on module only when needed.